addon/tests/helpers/with-feature.js

```diff
@@ -1,8 +1,8 @@
 import Ember from 'ember';
+import features from 'ember-feature-flags/features';
 
 export function withFeature( featureName ){
-  window.Features = window.Features || {};
-  window.Features[featureName] = true;
+  features.set(featureName, true);
```

Comment:
The intention of `withFeature` is to turn a feature on for a test or module (if called in `setup`). It can be called many times with different features. 

Re 1: There is a test helper called `resetFeatureFlags` that should be used to reset feature flags between tests. I find it convenient to add it to `startApp`.